task/06-conditions-n-loops-tasks.js

```diff
@@ -296,7 +331,24 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    var stack = [],
+        open = ['[', '(', '{', '<'],
+        close = [']', ')', '}', '>'];
+    
+    for(var i = 0; i < str.length; i++){
+        var comparer = str[i];
+
+        if( open.indexOf(comparer) !== -1 ){
+            stack.push(comparer);
+            continue;
+        }
+
+        var key = stack.pop();
+
+        if( open.indexOf(key) === close.indexOf(comparer) ){}
```

Comment:
It is better to use return of boolean expression instead of "if(expression) { /* empty */ } else {  }"